server/text_generation_server/utils/flash_attn.py

```diff
@@ -78,8 +89,25 @@ def attention(
             False,
             None,
         )
-
-    if HAS_FLASH_ATTN:
+    elif HAS_FLASH_ATTN_V2_ROCM:
+        # RoCm flash API does not take the window_size_left and window_size_right arguments.
```

Comment:
Then mistral is not supported or at least should trigger a warning that we are using the correct implementation.

Reply:
Oh that's true indeed, will fix!